src/solution.c

```diff
@@ -54,6 +54,10 @@ systime_t last_dgnss;
 
 double soln_freq = 10.0;
 u32 obs_output_divisor = 2;
+u32 gpgsv_msg_rate = 10;
+u32 gprmc_msg_rate = 10;
+u32 gpvtg_msg_rate = 10;
+u32 gpgll_msg_rate = 10;
```

Comment:
I'd be tempted to move these to `nmea.c`, `solution.c` is getting bloated as is.